tools/statistical_ensemble_test/README

```diff
@@ -1,56 +1,84 @@
-
-(2) If a new ensemble (and summary file) needs to be created, the
-ensemble.sh (or ensemble.py) script in this directory can first be used (see
-'ensemble.sh -h' for additional details) to create the ensemble. This
-script creates and compiles a case, then creates clones of the
-original case, with the pertlim in user_nl_cam is slightly modified
+$CESMDATAROOT/inputdata/validation/pop_ensembles
+
+If none of our ensembles are suitable for your needs, then you may create
+your own ensemble (and summary file) using the following instructions:
+     
+(1) To create a new ensemble, use the ensemble.py script in this directory. 
+This script creates and compiles a case, then creates clones of the
+original case, where the initial temperature perturbation is slightly modified
 for each ensemble member.  At this time, cime includes functionality
-to create ensembles for CAM-ECT and UF-CAM-ECT.
+to create ensembles for CAM-ECT, UF-CAM-ECT, and POP-ECT. 
 
-For example, for CAM-ECT:
+(2) Use --ect <pop.cam> to specify whether ensemble is for CAM or POP.
+(See 'python ensemble.py -h' for additional details).
 
-./ensemble.sh --case /glade/scratch/cesm_user/cesm_tag/ensemble/ensemble.cesm_tag.000 --mach cheyenne --compset F2000 --res f19_f19 --compiler intel --ensemble 151
+(3) Use --ensemble <size> to specify the ensemble size.
+Recommended ensemble sizes:
+CAM-ECT: 151
+UF-CAM-ECT: 350
+POP-ECT 40
 
-	      (or, replace "ensemble.sh" with "python ensemble.py")
+(4) Examples:
 
-For example, for UF-CAM-ECT:
+CAM-ECT:
+
+python ensemble.py --case /glade/scratch/cesm_user/cesm_tag/ensemble/ensemble.cesm_tag.000 --mach cheyenne   --ensemble 151 --ect cam --project P99999999
+
+
+UF-CAM-ECT:
+
+python ensemble.py --case /glade/scratch/cesm_user/cesm_tag/uf_ensemble/ensemble.cesm_tag.uf.000 --mach cheyenne  --ensemble 350 --uf --ect cam --project P99999999
+
+POP-ECT:
+
+python ensemble.py --case /glade/scratch/cesm_user/cesm_tag/uf_ensemble/ensemble.cesm_tag.000 --mach cheyenne  --ensemble 40 --ect pop --project P99999999
+
+Notes: 
+       (1) ensemble.py accepts (most of) the argumenets of create_newcase
+
+       (2) case name must end in ".000" and include the full path
 
-./ensemble.sh --case /glade/scratch/cesm_user/cesm_tag/uf_ensemble/ensemble.cesm_tag.uf.000 --mach cheyenne --compset F2000 --res f19_f19 --compiler intel --ensemble 350 --uf
+       (3) ensemble size must be specified, and suggested defaults are listed
+       	   above. Note that for CAM-ECT and UF-CAM-ECT, the ensemble size 
+	   needs to be larger than the number of variables that ECT will evaluate.
 
-	      (or, replace "ensemble.sh" with "python ensemble.py")
 
-Notes: -case name must end in ".000" and include the full path
-       -ensemble size must be specified, suggested defaults are 151 
-       for CAM-ECT and 350 for UF-CAM-ECT. Note that the ensemble size
-       needs to be larger than the number of variables that ECT will
-       evaluate.
+HERE
```

Comment:
Is this intentional?  It seems to be out of place.

Reply:
Oops - not intentional